File: Algorithm/Programmers/Pro42584.java
package com.algorithm.programmers;

public class Pro42584 {

	public static void main(String[] args) {
		int [] prices = {1, 2, 3, 2, 3};
		int [] answer = solution(prices);
        
        for(int ans : answer) {
        	System.out.print(ans+" ");
        }
	}
	

    static int[] solution(int[] prices) {
        int [] answer = new int[prices.length];
        
        int seconds = 0;
        int idx = 0;
        for(int i = 0 ; i < prices.length; i++){
            seconds = 0;
            for(int j = i+1; j < prices.length; j++){
                seconds += 1;
                if(prices[i] > prices[j]){
                    break;
                }
            }
            answer[idx++] = seconds;
        }        
        return answer;
    }

}

File: Python/Algorithm/Programmers/Pro42584.py
def solution(prices):
    answer = []

    for i in range(len(prices) - 1):
        seconds = 0  # 유지되는 시간
        for j in range(i + 1, len(prices)):
            if prices[i] <= prices[j]:
                seconds += 1
            else:
                seconds += 1
                break
        answer.append(seconds)
    answer.append(0)
    return answer

if __name__ == "__main__":
    prices = [1, 2, 3, 2, 3]
    print(solution(prices))